godotproject/Player.gd

```diff
@@ -260,7 +260,12 @@ func do_movement(delta):
 			# We decided that the frog should exit swinging state when they hit a wall or the ground
 			# Let's just make them stop swinging no matter what they collided with
 			stop_swing()
-
+			
```

Comment:
Can you add a comment here just stating what this chunk of code is for

Reply:
Can do!